src/citrine/resources/table_config.py

```diff
@@ -106,6 +107,13 @@ def __init__(self, name: str, *, description: str, datasets: List[UUID],
             raise ValueError("The data_source of the columns must match one of the variable names,"
                              " but {} were missing".format(missing_variables))
 
+    @property
+    def config_uid(self) -> UUID:
+        """[DEPRECATED] Unique ID of the table config, independent of its version."""
+        warnings.warn("The config_uid attribute is deprecated in favor of the uid attribute.", 
```

Comment:
Does the `deprecated` annotation work, as [here](https://github.com/CitrineInformatics/citrine-python/blob/main/src/citrine/resources/table_config.py#L269)?  I think that will play more nicely with the checking that @bfolie is trying to introduce in https://github.com/CitrineInformatics/citrine-python/pull/654